test/reducers/course.spec.js

```diff
@@ -1,16 +1,225 @@
 import deepFreeze from 'deep-freeze';
 import course from '../../app/assets/javascripts/reducers/course';
-import { UPDATE_COURSE } from '../../app/assets/javascripts/constants';
+import {
+  UPDATE_COURSE,
+  RECEIVE_COURSE,
+  RECEIVE_COURSE_UPDATE,
+  PERSISTED_COURSE,
+  CREATED_COURSE,
+  RECEIVE_INITIAL_CAMPAIGN,
+  ADD_CAMPAIGN,
+  RECEIVE_COURSE_CLONE,
+  DISMISS_SURVEY_NOTIFICATION,
+  TOGGLE_EDITING_SYLLABUS,
+  START_SYLLABUS_UPLOAD,
+  SYLLABUS_UPLOAD_SUCCESS,
+  LINKED_TO_SALESFORCE
+} from '../../app/assets/javascripts/constants';
 import '../testHelper';
 
 describe('course reducer', () => {
-  it('updates an attribute via UPDATE_COURSE', () => {
+  it('should return initial state when no action nor state is provided', () => {
+    const expectedState = {
+      title: '',
+      description: '',
+      school: '',
+      term: '',
+      level: '',
+      subject: '',
+      expected_students: '0',
+      start: null,
+      end: null,
+      timeline_start: null,
+      timeline_end: null,
+      day_exceptions: '',
+      weekdays: '0000000',
+      editingSyllabus: false
+    };
+
+    const newState = course(undefined, { type: null });
+    expect(newState).to.deep.eq(expectedState);
```

Comment:
This tests is tied to the exact form of the current initialState from the reducer. A better approach would just be to test for the presence of a few properties that you expect. In particular, something like expecting it to have keys for title and description, and maybe expect that weekdays is '0000000'. That will leave it open for extension if new properties get added to the initial state, which should not break the test.